pkg/manifests/configfile.go

```diff
@@ -84,6 +88,35 @@ func mustCompileConfigSchema() *jsonschema.Schema {
 	return schema
 }
 
+type Duration struct {
```

Comment:
I think `metav1.Duration` (k8s.io/apimachinery/pkg/apis/meta/v1) could be used here.

Reply:
I chose to create my own type so that I could marshal and unmarshal the duration.

Reply:
`metav1.Duration` has its own marshalling code, so it parses a value of e.g., `5m`. If it's a requirement to allow floats here, I guess you are stuck with implementing your own. Is that a hard requirement?

Reply:
Oh ok, perfect. I'm fine with no floats.